EXAMPLES.md

```diff
@@ -311,3 +312,59 @@ public IActionResult Admin()
     return View();
 }
 ```
+
+## Blazor Server
+
+The `Auth0-AspNetCore-Authentication` SDK works with Blazor Server in an almost identical way as how it's integrated in ASP.NET Core MVC.
+
+### Register the SDK
+Registering the SDK is identical as with ASP.NET Core MVC, where you should call `builder.Services.AddAuth0WebAppAuthentication` inside `Program.cs` and ensure to register the authentication middleware (`UseAuthentication()` and `UseAuthorization()`).
+
+```csharp
+builder.Services.AddAuth0WebAppAuthentication(options =>
+{
+    options.Domain = builder.Configuration["Auth0:Domain"];
+    options.ClientId = builder.Configuration["Auth0:ClientId"];
+    options.Scope = "openid profile email";
+});
+
+// ...
+
+var app = builder.Build();
+
+app.UseAuthentication();
+app.UseAuthorization();
+```
+
+### Adding Login and Logout
+Adding Login and Logout is different in the sense that you should create a `PageModel` implementation for both to allow the user to be redirected to and redirect further to Auth0.
```

Comment:
Looks like there were some extra words at the end that weren't needed, my suggestion:

```suggestion
Adding login and logout capabilities is different in the sense that you should create a `PageModel` implementation for both to allow the user to be redirected to Auth0.
```